src/vs/workbench/contrib/chat/browser/chatStatus.ts

```diff
@@ -129,7 +129,7 @@ export class ChatStatusBarEntry extends Disposable implements IWorkbenchContribu
 		this.registerListeners();
 	}
 
-	private update(): void {
+	private update = () => {
```

Comment:
I don't think we use this pattern very often. Could you use `() => this.update()` like other locations or use a real call to bind?